x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/default/flyout.tsx

```diff
@@ -61,18 +62,27 @@ export const DefaultDeprecationFlyout = ({
   return (
     <>
       <EuiFlyoutHeader hasBorder>
-        <EuiTitle size="s" data-test-subj="flyoutTitle">
-          <h2 id="defaultDeprecationDetailsFlyoutTitle" className="eui-textBreakWord">
-            {message}
-          </h2>
-        </EuiTitle>
-        {index && (
-          <EuiText data-test-subj="flyoutDescription">
-            <p>
-              <EuiTextColor color="subdued">{i18nTexts.getFlyoutDescription(index)}</EuiTextColor>
-            </p>
-          </EuiText>
-        )}
+        <EuiFlexGroup direction="column" gutterSize="s">
+          <EuiFlexItem>
+            <div>
```

Comment:
If we don't then the span will grow to fit the flex item, `flex-grow: 0` doesn't work also because of the context of where its rendered. We can either make the DeprecationBadge be wrapped around a div, have a custom class that adds a `margin-bottom` and remove the flexgroup/flexitems from the flyouts or this one with flexitems+wrapping div I've used here.

If we don't do any of that I think the badge ends up being too close to the title IMO, which is why I've added this in the first place:

![Screenshot 2021-09-13 at 09 41 11](https://user-images.githubusercontent.com/1191206/133259344-063a12d4-b61d-421d-b93f-59ee60184177.png)

Reply:
👍 [nit] Could you use `EuiSpacer`? Might be a little easier to follow. 

```
      <EuiFlyoutHeader hasBorder>

        <DeprecationBadge isCritical={deprecation.isCritical} isResolved={false} />

        <EuiSpacer size="s" />

        <EuiTitle size="s" data-test-subj="flyoutTitle">
          <h2 id="defaultDeprecationDetailsFlyoutTitle">{message}</h2>
        </EuiTitle>
        {index && (
          <EuiText data-test-subj="flyoutDescription">
            <p>
              <EuiTextColor color="subdued">{i18nTexts.getFlyoutDescription(index)}</EuiTextColor>
            </p>
          </EuiText>
        )}
      </EuiFlyoutHeader>
```